src/applications/personalization/profile360/vet360/actions/transactions.js

```diff
@@ -82,22 +82,24 @@ export function refreshTransaction(transaction, analyticsSectionName) {
 
       const transactionRefreshed = isVet360Configured() ? await apiRequest(`/profile/status/${transactionId}`) : await localVet360.updateTransaction(transactionId);
 
-      dispatch({
-        type: VET360_TRANSACTION_UPDATED,
-        transaction: transactionRefreshed
-      });
-
```

Comment:
The ultimate change here is that this only happens when the transaction is not successful.